packages/shared/src/styles/constants.scss

```diff
@@ -38,19 +41,35 @@ $color_grey-9: #868686;
 $color-grey-10: #919191;
 $color-grey-11: #fafafa;
 $color-grey-12: #f5f7fa;
+$color-grey-13: #979797;
+$color-grey-14: #b2c2c4;
+$color-grey-15: #a5a8a9;
+$color-grey-16: #c1cccf;
```

Comment:
Not related to this PR, But I don't understand the idea behind these SCSS variables, We just have 16 gray colors, How does having variables even help?! 😅

Reply:
@farzin-deriv Maybe some of them left from old design, I don't know :) we just agreed with @george-usynin-deriv to make these colors for Wallet and Wallet Modal to use the same colors in our PRs